packages/office-ui-fabric-react/src/components/Persona/Persona.scss

```diff
@@ -221,20 +222,20 @@ $ms-Persona-presenceSizeXl: 28px;
 }
 
 .ms-Persona-details {
+  box-sizing: border-box;
   padding: 0 12px;
   vertical-align: middle;
   @include text-align(left);
   @include padding-left($ms-Persona-sizeMd + $ms-Persona-imageDetailsLgSpace);
   display: table-cell;
+  width: 100%;
```

Comment:
This was recently [removed in another commit](https://github.com/OfficeDev/office-ui-fabric-react/commit/f3dbad4a2f852f8c9dc948aad71c51326f95ba89) by @joschect. Is it definitely necessary for text truncation to work?